dbdeployer_install.sh

```diff
@@ -123,4 +125,34 @@ then
   exit 1
 fi #end error check
 
+#create group
+if [ `grep "${group_name}" /etc/group | wc -l` -eq 0 ]
+then
+  groupadd "${group_name}"
+  if [ $? -ne 0 ]
+  then
+    echo "Failed to add group to system, exiting"
+    exit 1
+  fi #end error check
+fi
+
+#set permissions on log_dir
+chown root:${group_name} ${log_dir}
+if [ $? -ne 0 ]
+then
+  echo "Failed to set ownership of log directory, exiting"
+  exit 1
+fi #end error check
+chmod 775 ${log_dir}
+if [ $? -ne 0 ]
+then
+  echo "Failed to set permission on log directory, exiting"
+  exit 1
+fi #end error check
+
+#installation is complete
+echo "Installation has completed successfully."
 
+#warn that they will have to add users to the dbdeployer group before deploying files
+echo "Please add any users that will be deploying files to the 'dbdeployer' group."
```

Comment:
What will this change look like in Vagrant?  Or, how do you plan to communicate this change?